engine/src/main/scala/cromwell/engine/backend/BackendConfiguration.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell.engine.backend
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+import scala.collection.JavaConverters._
+
+/**
+  * Defines a backend configuration.
+  *
+  * @param name      Backend name.
+  * @param className Implementation class for the specific backend.
+  */
+case class BackendConfigurationEntry(name: String, className: String, config: Config)
```

Comment:
This `config` is only the chunk of the configuration specific to this backend instance.